.vscode/launch.json

```diff
@@ -1,14 +1,27 @@
 {
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
This is the version of the debug configuration file, not of the extension. I’m not 100% sure if this could technically be reverted, but I don’t really think it’s worth to try.

Reply:
Ahh okay. So when does it have to be increased?

Reply:
Whenever the format of `launch.json` is updated. The only reason it was `0.1.0`, is because it was old. When creating a new `launch.json` file from a template, it will use version `0.2.0`.